pyccel/codegen/printing/cucode.py

```diff
@@ -61,14 +70,107 @@ def _print_Module(self, expr):
             else:
                 local_imports += self._print(imp)
 
-        imports = f'{c_headers_imports}\
-                    extern "C"{{\n\
-                    {local_imports}\
-                    }}'
+        imports = f'{c_headers_imports}\n{local_imports}'
```

Comment:
Can you rename `c_headers_imports` so it is clear that it contains cuda imports not C imports please